### PR TITLE
Minimal Touch app that simply waits for a touch while flashing the LED.

### PR DESCRIPTION
**Pull request by dehanj**
Intended to be both an example for others and useful in production while testing the touch functionality. Hence I wrote it so one could quickly test multiple touches in a row, without the need of a host app. 

The app waits for a touch, confirms by changing the flashing color. 
Then waits for a new one and so on, for eternity.

### CODE REVIEW

File: apps/touch/main.c [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+void wait_touch_ledflash(int ledvalue, int loopcount)
+{
+	int led_on = 0;
+	// first a write, to ensure no stray touch?
+	*touch = 0;
+	for (;;) {
+		*led = led_on ? ledvalue : 0;
+		for (int i = 0; i < loopcount; i++) {
+			if (*touch & (1 << TK1_MMIO_TOUCH_STATUS_EVENT_BIT)) {
+				goto touched;
```
Comment (secworks) [inline apps/touch/main.c:26]: goto? Ouch!

Comment (mchack-work) [reply apps/touch/main.c:26]: This is just copy/paste from the signer https://github.com/tillitis/tillitis-key1-apps/blob/main/apps/signer/main.c#L26

I think use of goto is OK if it's used forward-only to break out of nested loops or so.

**Review by mchack-work — CHANGES_REQUESTED** [review]
Fix linebreaks, then I can approve.

File: README.md [hunk 2]
```diff
@@ -445,6 +445,17 @@ In `blink/` there is also a very, very simple app written in
 assembler, `blink.bin` (blink.S) that blinks the LED.
 
 
+##  Example Touch app
+
+In `touch/` resides an example app of how to use the built in touch feature of the TKey. The application simply waits for a touch from the user while flashing the LED. A touch is confirmed by switching the flashing color while starting to wait for a new touch - shifting colors between green, red and blue.
```
[the remark this answers is not in the document]
Comment (dehanj) [reply README.md]: Apparently VScode extension for editorconfig does not have support for just `max_line_length`. I'll have to find a workaround. But I fixed the line breaks.

File: apps/touch/main.c (hunk 1 above)
Comment (dehanj) [reply apps/touch/main.c:26]: Yeah, just a copy from signer. Didn't prioritize to rewrite it.

**Comment by mchack-work** [comment]
Squashed and merged.

**Pull request closed without merging**